src/js/controller/model.js

```diff
@@ -381,6 +388,19 @@ define([
             }
         };
 
+
+        this.setPlaybackRate = function(playbackRate) {
+            if (!_provider || !_.isNumber(playbackRate) || this.get('streamType') === 'LIVE' || !_attached) {
```

Comment:
Yeah, we can attempt to set it on the provider if we have a provider, otherwise just set it up our intended playback rate value in the model if not.